dali2mqtt/lamp.py

```diff
@@ -90,6 +90,13 @@ def level(self):
     @level.setter
     def level(self, value):
         """Commit level to ballast."""
+
+        if not isinstance(value, (int, float)):
+            if value == "MASK":
+                raise ValueError("Lights are in pre-heating mode. Call this implementation a bit later.")
+            else:
+                raise ValueError(f"value should not be {value}")
```

Comment:
```suggestion
                raise ValueError("value should not be %s", value)
```

Reply:
You don't like lazy formatting? I see it also in your original code and i think it is more transparent.

Reply:
I'm trying to align the code style of this project with that of Home Assistant, in which f-strings are banned in logging. Please update